### ISSUE TITLE
translation for palletes "Arpeggios & glissandi" and "fingering" are missing in the app

### ISSUE THREAD

**Issue by Yanko64** [musescore/MuseScore#20766]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

even when they are translated in Transifex, the translations still don't show up in the app (for example, in Hebrew)

### Steps to reproduce

1. update the translation in any language 
2. open the app
3. see that the translations are missing

### Screenshots/Screen recordings

![image](https://github.com/musescore/MuseScore/assets/65133734/7de31d03-af2a-4854-9c31-0a8eda3f810c)

### MuseScore Version

4.2.0

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 10

**Comment by Jojo-Schmitz** [comment]
Revert to factory settings, IIRC to have had the same issue, and a reset fixing it

**Comment by Jojo-Schmitz** [comment]
There might be a bug lurking though: these leftovers (from previous versions) shouldn't really happen

**Comment by manolo** [comment]
The same happening in Spanish
![Screenshot 2024-01-25 at 14 52 35](https://github.com/musescore/MuseScore/assets/161853/bde95054-3646-4f1d-aa42-f9ae684a936b)

Even though translation [site](https://app.transifex.com/musescore/musescore/translate/#es/musescore/430527715?q=text%3AFingering) seems to have translation for those terms
![Screenshot 2024-01-25 at 14 52 58](https://github.com/musescore/MuseScore/assets/161853/12a5253b-830e-453e-9b8e-ddf750e36926)

**Comment by Jojo-Schmitz** [comment]
Use Help > Revert to factory settings, those entries are leftovers
Same issue came up today in https://musescore.org/en/node/359695

**Comment by zacjansheski** [comment]
@Yanko64 did a revert to factory settings resolve this issue?

**Comment by Jojo-Schmitz** [comment]
Even if it did (which I'm preztzty sure about) theis still is an issue, such a revert to factotory setting shopuld not be needed after a minor version update

**Comment by zacjansheski** [comment]
If I open 4.3 and change the language to Hebrew/Spanish, the correct translations appear without any need for factory reset.
@DmitryArefiev we will want to double check in next RC

**Comment by DmitryArefiev** [comment]
@zacjansheski Yes, but I think we don't have a full steps for now..  I can't reproduce the bug even in 4.2.0/4.2.0->4.2.1 (Windows10)

**Comment by DmitryArefiev** [comment]
OK. I found the steps. That occurs when updating from the old versions of MuseScore 4 (e.g. **4.0.0 -> 4.2/4.2.1/4.3**)

1. Uninstall MS
2. Delete **c:\Users\user\AppData\Local\MuseScore\MuseScore4** and **c:\Users\user\AppData\Roaming\MuseScore\MuseScore4.ini**
3. Install **4.0.0**
4. Switch to Hebrew
5. Restart MS
6. Add all palettes
7. In Preferences, Check for language updates
8. Restart MS
9. All palettes translated fine
10. Update Musescore to **4.2.1**
11. Run MS
12. All palettes translated fine
13. In Preferences, Check for language updates
14. Restart MS
15. "Arpeggios & glissandi" and "fingering" are no translated

https://github.com/musescore/MuseScore/assets/90187801/adbe35d5-5156-4202-9b8f-87eaa0fff0a8

**Comment by Jojo-Schmitz** [comment]
Happens again/still in 4.3.0, see #22587

**Comment by Jojo-Schmitz** [comment]
And still with 4.4, current master, came up on Discord, https://discord.com/channels/818804595450445834/818852942882275379/1263488653846511750
